examples/misc/lib/language_tour/misc/for_loop_in_collection_a.dart

```diff
@@ -0,0 +1,8 @@
+void main() {
+  // #docregion code_sample
+  var numbers = [2, 3, 4];
+  var items = [1, for (var n in numbers) n * n, 7]; // [1, 4, 9, 16, 7]
```

Comment:
Since you want to include the output, you could include the `print` calls in you snippets as well and put the comments before that.

Then developers could quickly copy and paste the whole snippets to try themselves as well.

Another option is to structure these as tests and use the `equals` matchers. Then we could also run these tests in CI to know the output is correct.

```dart
var numbers = [2, 3, 4];
var items = [1, for (var n in numbers) n * n, 7];

expect(items, equals([1, 4, 9, 16, 7]));
```